### ISSUE TITLE
BUG: reinitialization ContextMenu after redraw map

### ISSUE THREAD

**Issue by ollazarev** [jonataswalker/ol3-contextmenu#51]
257, 266 lines in _ol3-contextmenu-debug.js_:

```
257: var topics = {};
...
266: var index = topics[topic].push(listener) -1;
```

Variable _'topics'_ is not redefined, when ContextMenu is reinitialized. So item's callbacks are accumulated and called multiply, when the user reinitialize ContextMenu after redrawing the map.

**Comment by jonataswalker** [comment]
@ollazarev  Are you able to provide a fiddle to reproduce this bug?

**Comment by ollazarev** [comment]
http://jsfiddle.net/gLmq75sp/

**Comment by jonataswalker** [comment]
Fixed with #53 - @ollazarev  thanks for reporting.
